R/sql-build.R

```diff
@@ -61,15 +61,15 @@ sql_build.op_rename <- function(op, con, ...) {
 
 #' @export
 sql_build.op_arrange <- function(op, con, ...) {
-  order_vars <- translate_sql_(op$dots, con)
+  order_vars <- translate_sql_(op$dots, con, sql_context = list(arrange = TRUE))
```

Comment:
I think this would all be simpler if used one variable name everywhere - i.e. `sql_context = list(op = "arrange")`